coderd/database/dbauthz/authzquerier.go

```diff
@@ -0,0 +1,103 @@
+package dbauthz
```

Comment:
This file should be renamed to `dbauthz.go` and the general file in the package.

Reply:
Ahh, I see there's already a `dbauthz.go`. Might be cleaner to put this in there... it's odd that the `New` func for the package isn't in the same filename.

Reply:
Fair. We just need a place to move all those methods too. We merged all the methods into 1 file, just make naming that file a little annoying. I could just name it `methods.go`

Reply:
Consolidated into `dbauthz.go` and moved the other files to `methods.go`